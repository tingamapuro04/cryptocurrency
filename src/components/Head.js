import React from 'react';
import styles from './cryptolist.module.css';
import head from '../img/cryptoimg1.jpeg';

function Head() {
  return (
    <div className={styles.headdiv}>
      <img alt="Head pic" className={styles.Headimage} src={head} />
    </div>
  );
}

export default Head;
