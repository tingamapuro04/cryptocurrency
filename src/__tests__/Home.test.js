// import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import Head from '../components/Head';

it('Renders correctly', () => {
  const tree = renderer.create(<Head />).toJSON();
  expect(tree).toMatchSnapshot();
});
