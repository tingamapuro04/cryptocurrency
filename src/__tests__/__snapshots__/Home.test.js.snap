// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="headdiv"
>
  <img
    alt="Head pic"
    className="Headimage"
    src="Trade.jpeg"
  />
</div>
`;
